src/Compilers/CSharp/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// The nullable annotations that can apply in source.
+    /// </summary>
+    internal enum NullableAnnotation : byte
```

Comment:
Consider adding a comment that order is important here as the code base will do logic comparisons on the numeric value.